gcloud/datastore/test_connection.py

```diff
@@ -520,7 +520,7 @@ def test_run_query_wo_eventual_w_transaction(self):
         DATASET_ID = 'DATASET'
         KIND = 'Nonesuch'
         CURSOR = b'\x00'
-        TRANSACTION = 'TRANSACTION'
+        TRANSACTION = b'TRANSACTION'
```

Comment:
Why don't other strings like `DATASET_ID` need this treatment? Is there a "best-practice" about naked `'foo'` which end up being different types in Python 2 and 3?